### PR TITLE
Retrieve test package url via mozdownload if it's not available (#738)

### PR DESCRIPTION
**Pull request by whimboo**
This PR will fix issue #738. It's not ready to be reviewed yet. I might still have to do some tweaks and documenation.

### CODE REVIEW

**Comment by whimboo** [comment]
Here an example for retrieving the non-prefixed test package URL for the 45.0b5 candidate build, and triggering a Jenkins job:

> 2016-02-22T22:18:20Z  INFO mozmill-ci: firefox 45.0b5 (201602120925, 66afdb3943e998d1a1a5aab1d92eb9663179bcd6, ar, linux64) [mozilla-beta]
2016-02-22T22:18:20Z  INFO mozmill-ci: Querying installer URL...
2016-02-22T22:18:21Z  INFO mozmill-ci: Found installer at: https://archive.mozilla.org/pub/firefox/candidates/45.0b5-candidates/build1/linux-x86_64/ar/firefox-45.0b5.tar.bz2
2016-02-22T22:18:21Z  INFO mozmill-ci: Querying tinderbox revision for release-mozilla-beta build...
2016-02-22T22:18:24Z  INFO mozmill-ci: Found revision for tinderbox build: bf3681b5be9a4fbeb525475e8197833b0bc20b8b
2016-02-22T22:18:24Z  INFO mozmill-ci: Querying test packages URL...
2016-02-22T22:18:29Z  INFO mozmill-ci: URL not found. Querying not-prefixed test packages URL...
2016-02-22T22:18:35Z  INFO mozmill-ci: Found test package URL at: https://archive.mozilla.org/pub/firefox/tinderbox-builds/mozilla-beta-linux64/1455224373/test_packages.json
2016-02-22T22:18:35Z  INFO mozmill-ci: Triggering job "release-mozilla-beta_functional" on "linux && ubuntu && 14.04 && 64bit"

File: config/dev/pulse.json [hunk 1]
```diff
@@ -6,12 +6,9 @@
             "mozilla-central": {
                 "blacklist": {
                     "locales": [
-                        "fr"
```
Comment (whimboo) [inline config/dev/pulse.json]: I removed those blacklist entries to make local testing easier.

**Comment by whimboo** [comment]
@mjzffr and  @sydvicious this PR is ready for your review now. Thanks.

**Comment by sydvicious** [comment]
@whimboo r+

**Comment by whimboo** [comment]
@mjzffr also had a look at this PR and gave me her r+ via IRC. So thank you both! I will get this PR landed in a bit.

**Comment by whimboo** [comment]
This PR was merged to master as https://github.com/mozilla/mozmill-ci/commit/71e8ff70e417ed6fdcdee3c1994adb82a7aa5dcc.

**Pull request closed without merging**